tests/libtest.sh

```diff
@@ -1,5 +1,18 @@
 #!/bin/bash
 
+# Test if config exists.
+config_exists(){
+  local config_dir="/var/lib/container-storage-setup"
```

Comment:
Also we will require similar functionality in container-storage-setup.sh as well to make sure configuration does not exist. May be we should start sourcing libcss.sh also. Any functions common to both container-storage-setup.sh and tests should be put in libcss.sh and anything local to tests only should be put in libtest.sh.